src/Tizen.System.PowerUsage/PowerUsage/PowerUsage.cs

```diff
@@ -0,0 +1,242 @@
+        /// <param name="start">Start Time for data in DateTime.</param>
+        /// <param name="end">End Time for data in DateTime.</param>
+        /// <returns>Returns the total battery usage in mAh(milli-Ampere hour) by an application for certain time interval.</returns>
+        /// <feature>http://tizen.org/feature/battery</feature>
+        /// <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+        /// <exception cref="ArgumentException">When an invalid parameter value is set.</exception>
+        /// <exception cref="UnauthorizedAccessException">If the privilege is not set.</exception>
+        /// <exception cref="InvalidOperationException">In case of any system error.</exception>
+        /// <exception cref="NotSupportedException">In case power usage is not supported</exception>
+        public static double GetPowerUsage(string appID, DateTime start, DateTime end)
+        {
+            double batteryUsage = 0;
+            DateTimeOffset startTime = DateTime.SpecifyKind(start, DateTimeKind.Utc);
+            DateTimeOffset endTime = DateTime.SpecifyKind(end, DateTimeKind.Utc);
+            PowerUsageError ret = (PowerUsageError)Interop.PowerUsage.GetPowerUsageByApp(appID, startTime.ToUnixTimeSeconds(), endTime.ToUnixTimeSeconds(), out batteryUsage);
+            if (ret != PowerUsageError.None)
+            {
+                Log.Error(PowerUsageErrorFactory.LogTag, "Error getting battery usage by app," + ret);
+                throw PowerUsageErrorFactory.ThrowPowerUsageException(ret);
+            }
+            return batteryUsage;
+        }
+
+        /// <summary>
+        /// Gets the battery usage in mAh(milli-Ampere hour) by a resource for certain time interval.
+        /// </summary>
+        /// <since_tizen> 7 </since_tizen>
+        /// <param name="rtype">Identifier of resource type. BLE, WiFi, CPU etc.</param>
+        /// <param name="start">Start Time for data in DateTime.</param>
+        /// <param name="end">End Time for data in DateTime.</param>
+        /// <returns>Returns the battery usage in mAh(milli-Ampere hour) by a resource for certain time interval.</returns>
+        /// <feature>http://tizen.org/feature/battery</feature>
+        /// <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+        /// <exception cref="ArgumentException">When an invalid parameter value is set.</exception>
+        /// <exception cref="UnauthorizedAccessException">If the privilege is not set.</exception>
+        /// <exception cref="InvalidOperationException">In case of any system error.</exception>
+        /// <exception cref="NotSupportedException">In case power usage is not supported</exception>
+        public static double GetPowerUsage(PowerUsageResourceType rtype, DateTime start, DateTime end)
+        {
+            double batteryUsage = 0;
+            DateTimeOffset startTime = DateTime.SpecifyKind(start, DateTimeKind.Utc);
```

Comment:
Why do you convert to Utc?

I have test
``` c#
            var now = DateTime.Now;
            var utc_now = DateTime.UtcNow;
            DateTimeOffset converted = DateTime.SpecifyKind(now, DateTimeKind.Utc);

            Console.WriteLine($"Now epoch time {((DateTimeOffset)now).ToUnixTimeSeconds()}");
            Console.WriteLine($"utc_now epoch time {((DateTimeOffset)utc_now).ToUnixTimeSeconds()}");
            Console.WriteLine($"converted epoch time {converted.ToUnixTimeSeconds()}");
```
``` 
Now epoch time 1586480128
utc_now epoch time 1586480128
converted epoch time 1586512528
```

Reply:
Right, Since user doesn't send UTC time as input, I should either change SpecifyKind to local or there is no need to SpecifyKind seperately.

I tested on this

var now = DateTime.Now;
            var utc_now = DateTime.UtcNow;
            DateTimeOffset converted = DateTime.SpecifyKind(now, DateTimeKind.Local);

            Console.WriteLine($"Now epoch time {((DateTimeOffset)now).ToUnixTimeSeconds()}");
            Console.WriteLine($"utc_now epoch time {((DateTimeOffset)utc_now).ToUnixTimeSeconds()}");
            Console.WriteLine($"now.ToUniversalTime epoch time {((DateTimeOffset)now.ToUniversalTime()).ToUnixTimeSeconds()}");
            Console.WriteLine($"converted epoc time {converted.ToUnixTimeSeconds()}");

Now epoch time 1586722925
utc_now epoch time 1586722925
now.ToUniversalTime epoch time 1586722925
converted epoc time 1586722925